packages/psss/src/containers/Panels/AlertsPanel.js

```diff
@@ -70,60 +69,72 @@ const menuOptions = [
 
 const TabsContext = React.createContext(null);
 
-export const AlertsPanel = React.memo(({ isOpen, handleClose }) => {
-  const [isModalOpen, setIsModalOpen] = useState(false);
-  const [activeIndex, setActiveIndex] = useState(0);
-  const sitesState = useFetch(getAffectedSites);
-  const notesState = useFetch(getAlertsMessages);
-  const activityState = useFetch(getActivityFeed);
-  const { countryCode } = useParams();
-  const countryName = useSelector(state => getCountryName(state, countryCode));
+export const AlertsPanel = React.memo(
```

Comment:
Apologies for the diff in this file - a lot of it is due to nesting change by `prettier`